_inc/lib/core-api/wpcom-endpoints/service-api-keys.php

```diff
@@ -121,7 +121,10 @@ public static function get_service_api_key( $request ) {
 
 		switch ( $service ) {
 			case 'mapbox':
-				$mapbox                 = self::get_service_api_key_mapbox();
+				if ( ! class_exists( 'Jetpack_Mapbox_Helper' ) ) {
+					jetpack_require_lib( 'class-jetpack-mapbox-helper' );
+				}
+				$mapbox                 = Jetpack_Mapbox_Helper::get_access_token();
```

Comment:
That's how the Service API Keys endpoint has been working all along though.

Here's before I touched it:

https://github.com/Automattic/jetpack/blob/75d961890019318654875e7e57016152e42de562/_inc/lib/core-api/wpcom-endpoints/service-api-keys.php#L118-L124

Note that the message is always "API key retrieved successfully.", regardless of the actual API key value: `'service_api_key' => Jetpack_Options::get_option( $option, '' )`

I guess the message means that the _Service API Keys request_ was successful, rather than the _API key_ existing.

Reply:
Yeah, that's a small detail. Maybe worth updating the message? It's not that important reall.